libbeat/common/field.go

```diff
@@ -99,6 +99,44 @@ func (f Fields) HasKey(key string) bool {
 	return f.hasKey(keys)
 }
 
+// HasNode checks if inside fields the given node exists
+// In contrast to HasKey it not only compares the leave nodes but
```

Comment:
`s/leave/leaf/` since it's singular in this context.

Reply:
fixed